html/webappapis/system-state-and-capabilities/the-navigator-object/protocol.https.html

```diff
@@ -31,38 +31,38 @@
   location.href + '/%s/bar/baz/',
   location.href + '/%s/bar/baz/?foo=1337&bar#baz',
   location.href + '/%s/foo/%s/',
-];
-for (const url of valid_urls) {
+].forEach(url => {
   test(() => {
     navigator.registerProtocolHandler('tel', url, 'foo');
   }, 'registerProtocolHandler: Valid URL "' + url + '" should work.');
 
   test(() => {
     navigator.unregisterProtocolHandler('tel', url);
   }, 'unregisterProtocolHandler: Valid URL "' + url + '" should work.');
-}
+});
 
-const invalid_urls1 = [
+/* Invalid URLs */
+[
   '',
   '%S',
   location.href + '',
   location.href + '/%',
   location.href + '/%a',
   'http://example.com',
   'http://[v8.:::]//url=%s',
-];
-
-for (const url of invalid_urls1) {
+].forEach(url => {
   test(() => {
     assert_throws_dom('SYNTAX_ERR', () => { navigator.registerProtocolHandler('mailto', url, 'foo'); });
-  }, 'registerProtocolHandler: Invalid URL "' + url + '" should throw SYNTAX_ERR.');
+    assert_throws_dom('SECURITY_ERR', () => { navigator.registerProtocolHandler('x', url, 'foo'); });
+  }, 'registerProtocolHandler: Invalid URL "' + url + '" should throw (but after scheme)');
```

Comment:
Just nits; it seems a good chance to take advantage of the new ES syntax.

```suggestion
  }, `registerProtocolHandler: Invalid URL "${url}" should throw (but after scheme)`);
```